### PR TITLE
chore(VSCode): publish extension to Open VSX

### PR DESCRIPTION
**Pull request by sebmck**
## Summary

Adds publishing to [Open VSX](https://open-vsx.org/) to our release actions.

## Test Plan

Not sure? cc @ematipico @leops for how I might test this. If the changes look okay then we can wait for the next release before publishing our first version.

### CODE REVIEW

File: .github/workflows/release_lsp.yml [hunk 1]
```diff
@@ -196,17 +196,28 @@ jobs:
...
           VSCE_PAT: ${{ secrets.VSCE_PAT }}
-      - name: Publish extension
+      - name: Publish extension to Microsoft Marketplace
         run: npx vsce publish --packagePath rome_lsp-*.vsix
         if: needs.build.outputs.prerelease != 'true'
         env:
           VSCE_PAT: ${{ secrets.VSCE_PAT }}
 
+      - name: Publish extension to Open VSX (pre-release)
+        run: npx vsce publish --pre-release --packagePath rome_lsp-*.vsix
```
Comment (MichaReiser) [inline .github/workflows/release_lsp.yml:211]: The command must be `ovsx` according to the documentation.

```
npx ovsx publish <file> -p <token>
```

We probably need to add it as a dependency too

Comment (sebmck) [reply .github/workflows/release_lsp.yml:211]: Oh oops. Adding it as a dependency isn't necessary as `npx` will install if it isn't already.

**Comment by sebmck** [comment]
Fixed binary reference. I've also added the Open VSX PAT token to the `marketplace` environment so this should just work.

**Comment by MichaReiser** [comment]
Looks good. Can you manually trigger an LSP workflow and then approve it to publish a nightly release to the marketplace? 

https://github.com/rome/tools/actions/workflows/release_lsp.yml

**Review by MichaReiser — CHANGES_REQUESTED** [review]
Please update the test plan with a run of the LSP action.

**Comment by ematipico** [comment]
What's the documentation plan of this PR? I assume this needs to be documented somewhere on our website

**Comment by MichaReiser** [comment]
> What's the plan to document this? I assume this needs to be documented somewhere on our website

We should mention it in the VS Code section

**Comment by ematipico** [comment]
> > What's the plan to document this? I assume this needs to be documented somewhere on our website
> 
> We should mention it in the VS Code section

Sorry, I actually meant the real plan. There wasn't any issue to track documentation changes, so I wanted to know where/how we take note of this new change.

**Comment by MichaReiser** [comment]
Test plan: https://github.com/rome/tools/actions/runs/3477745546/jobs/5814546845

**Review by MichaReiser — APPROVED** [review]

**Pull request merged**